components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -75,11 +78,15 @@ This table lists the [configurations of components](/components/#creating-compon
 | `kbc_project_id` | Foreign key to the Keboola project | `580_kbc-us-east-1` |
 | `kbc_component_id` | Identifier of the component | `keboola.python-transformation-v2` |
 | `kbc_component` | Name of the component | `Generic` |
+| `kbc_component_type` | Identifier of the component type | `extractor` |
+| `kbc_component_vendor` | Identifies who created the component | `keboola` |
```

Comment:
@lucie-krasna Tady u dvou řádků je `identifies`, ale standardně používáme `identifier` jako podstatné jméno - opět bych se inspiroval dalšími identifikátory. Pokud narazíš na nekonzistenci u existujících řádků, klidně to oprav i tam.